src/components/ButtonComponents/OperatorButtons/Operators.js

```diff
@@ -1,16 +1,26 @@
-import React from "react";
+import React, { useState } from "react";
 
 //import any components needed
+import OperatorButton from './OperatorButton';
+import { ButtonsContainer } from '../SpecialButtons/Specials';
 
 //Import your array data to from the provided data file
+import { operators } from '../../../data';
 
-const Operators = () => {
+const Operators = props => {
   // STEP 2 - add the imported data to state
+  const [calcOperators, setCalcOperators] = useState(operators);
+
   return (
-    <div>
+    <ButtonsContainer>
       {/* STEP 3 - Use .map() to iterate over your array data and return a button
        component matching the name on the provided file. Pass
        it any props needed by the child component*/}
-    </div>
+       {calcOperators.map(operator => 
+          <OperatorButton key={operator.char} operator={operator.char} operation={props.operation} />
```

Comment:
Earlier when i was talking about the if conditional you made, this is how it would play out...
Step 1: instead of doing `operator={operator.char}`, you would pass in the whole object: `operator={operator}